.github/workflows/build_test.yml

```diff
@@ -0,0 +1,109 @@
+name: Standard Build and Test
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
```

Comment:
Do we want to limit which branches this triggers for `push`

Reply:
we could probably limit them to develop...
and open the PR to all branches ?

Reply:
Doing it on push for all branches, means every push to my own branch on my own repo will trigger an action (for better or worse) - right?

Reply:
yes

Reply:
we don't I think requesting a PR would allow to limit the extend of it.
if one needs the CI, one can always issue a PR in a fork